examples/getting-started-movielens/04-Triton-Inference-with-TF.ipynb

```diff
@@ -35,26 +35,7 @@
     "\n",
```

Comment:
<p>Please remove this code <code>docker run -it --gpus device=0 ..</code> line .   this is no longer the case. we are in the <code>merlin-tensorflow</code> container which does serving as well. so  no need to launch a new container. Please remove the sentence <code>Before we get started, you should start the container for Triton Inference Server with the following command. This command includes the -v argument that mounts your local model-repository folder with your saved models from the previous notebook (03-Training-with-TF.ipynb).</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>done</p><p> </p>